### PR TITLE
Add contracts ContenType to golang/elixir facade

### PR DESCRIPTION
**Pull request by CDimonaco**

### CODE REVIEW

File: golang/pkg/events/events.go [hunk 1]
```diff
@@ -63,3 +63,8 @@ func EventType(src []byte) (string, error) {
 	name := any.ProtoReflect().Type().Descriptor().FullName()
 	return string(name), nil
 }
+
+// ContentType returns the content type of used contracts
+func ContentType() string {
+	return "application/x-protobuf"
+}
```
Comment (fabriziosestito) [inline golang/pkg/events/events.go:70]: Just curious, why not an exported constant ?

**Review by fabriziosestito — APPROVED** [review]

**Review by arbulu89 — APPROVED** [review]
Are we going to use this for something more than informative reasons?

File: golang/pkg/events/events.go (hunk 1 above)
Comment (CDimonaco) [reply golang/pkg/events/events.go:70]: No strong feelings here, I was just thinking about the api, if in the future we need to perform more task to get the content type

**Comment by CDimonaco** [comment]
> Are we going to use this for something more than informative reasons?

The idea is to use that as rabbitmq content type, the elixir library by default put application/json, so we need to modify that, and we have the same also in golang

**Pull request merged**